pkg/deploy/strategy/rolling/rolling.go

```diff
@@ -205,16 +205,22 @@ func (s *RollingDeploymentStrategy) Deploy(from *kapi.ReplicationController, to
 		UpdatePeriod:   time.Duration(*params.UpdatePeriodSeconds) * time.Second,
 		Interval:       time.Duration(*params.IntervalSeconds) * time.Second,
 		Timeout:        time.Duration(*params.TimeoutSeconds) * time.Second,
+		UpdatePercent:  params.UpdatePercent,
 		CleanupPolicy:  kubectl.PreserveRollingUpdateCleanupPolicy,
 		UpdateAcceptor: updateAcceptor,
 	}
-	glog.Infof("Starting rolling update from %s to %s (desired replicas: %d, UpdatePeriodSeconds=%d, IntervalSeconds=%d, TimeoutSeconds=%d)",
+	pct := "<nil>"
+	if params.UpdatePercent != nil {
+		pct = fmt.Sprintf("%d", *params.UpdatePercent)
+	}
+	glog.Infof("Starting rolling update from %s to %s (desired replicas: %d, updatePeriodSeconds=%d, intervalSeconds=%d, timeoutSeconds=%d, updatePercent=%s)",
```

Comment:
Add a *%* in the end of updatePercent? Also *s* in the end of *Seconds would be nice.